src/components/ModalWindow/ModalWindow.tsx

```diff
@@ -49,7 +64,7 @@ const ModalWindow: FC = () => {
             <Form.Control
               className={s.formControl}
               type="text"
-              value={input.message}
+              value={`${input.message === "undefined" ? "" : input.message}`}
```

Comment:
Try to create better condition

Reply:
I try, don't have any options how i can better change this.

Reply:
Try something with !!

Reply:
During the destructuring fix, I understood why the 'undefined' string came, fixed it, now there is no need to check in value